packages/cf-style-const/src/variables.js

```diff
@@ -97,5 +97,5 @@ export default {
   colorFacebookBlue: '#3b5998',
   colorMainBackground: '#e6e6e6',
   zIndexMax: 1000,
-  modalBoxShadow: '0 0 0 9999px rgba(0, 0, 0, .7)'
+  modalBackdropBackground: 'rgba(0, 0, 0, .7)'
```

Comment:
You guys use `background: #000; opacity: .7` for your modal backdrop. So perhaps you’ll want to copy those two constants from there.